std/internal/math/gammafunction.d

```diff
@@ -548,7 +550,11 @@ unittest {
 
 
 private {
-static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended) {
+static if (floatTraits!(real).realFormat == RealFormat.ieeeQuadruple) {
+    enum real MAXLOG = 0x1.62e42fefa39ef35793c7673007e6p+13;  // log(real.max)
+    enum real MINLOG = -0x1.6546282207802c89d24d65e96274p+13; // log(real.min_normal*real.epsilon) = log(smallest denormal)
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/Quadruple-precision_floating-point_format) and the [gnu's libquadmath](https://github.com/gcc-mirror/gcc/blob/master/libquadmath/quadmath.h#L145) the max value with quadruple precision is 
`1.18973149535723176508575932662800702e4932`, [thus](http://www.wolframalpha.com/input/?i=log(1.18973149535723176508575932662800702e4932)):

```
11356.52340629414394949193107797076489126
````

afaict the current representation has less digits?

real.min is `3.36210314311209350626267781732175260e-4932` and [thus](http://www.wolframalpha.com/input/?i=log(3.36210314311209350626267781732175260e-4932)) 

```
-11355.13711193302405887309661372784853821
```